dali/pipeline/operators/reader/reader_op_test.cc

```diff
@@ -119,4 +119,50 @@ TYPED_TEST(ReaderTest, SimpleTest) {
   return;
 }
 
+
+TYPED_TEST(ReaderTest, SequenceTest) {
+  Pipeline pipe(128, 4, 0);
+
+  pipe.AddOperator(
+      OpSpec("SequenceReader")
+      .AddArg("file_root", image_folder + "/frames/")
```

Comment:
Is `image_folder` defined somwhere? Looks like compilation error

Reply:
dali/test/dali_test.h:37

Reply:
Oh, tricky! I basically forgot, that this is a macro, which shares variables with the GTest-proper class defined above (also, it'd not really clear at the first glance, that this is GTest-proper class...). Thus, I started to looking somewhere for definition of `image_folder` inside the file... Could you make in `this->image_folder`? It would be much more meaningful.

Reply:
I can't, this is a global variable.